src/node_process_object.cc

```diff
@@ -229,28 +233,25 @@ void PatchProcessObject(const FunctionCallbackInfo<Value>& args) {
   SECURITY_REVERSIONS(V)
 #undef V
 
-  // process.execPath
-  process
-      ->Set(context,
-            FIXED_ONE_BYTE_STRING(isolate, "execPath"),
-            String::NewFromUtf8(isolate,
-                                env->exec_path().c_str(),
-                                NewStringType::kInternalized,
-                                env->exec_path().size())
-                .ToLocalChecked())
-      .Check();
+  if (!ToV8Value(context, env->exec_path(), isolate).ToLocal(&val) ||
```

Comment:
Same, better to keep the comment for searching.

```suggestion
  // process.execPath
  if (!ToV8Value(context, env->exec_path(), isolate).ToLocal(&val) ||
```